service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/documents/ManageDocumentsLAController.java

```diff
@@ -207,14 +229,21 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(@RequestBod
                     caseDetailsMap, sortedDocuments, CORRESPONDING_DOCUMENTS_COLLECTION_LA_KEY
                 );
                 caseDetailsMap.putIfNotEmpty(CORRESPONDING_DOCUMENTS_COLLECTION_LA_KEY, sortedDocuments);
+                caseDetailsMap.put(DOCUMENT_WITH_CONFIDENTIAL_ADDRESS_KEY,
+                    manageDocumentService.getDocumentWithConfidentialAddress(caseData,
+                        caseData.getCorrespondenceDocumentsLA(),
+                        sortedDocuments));
                 break;
             case ADDITIONAL_APPLICATIONS_DOCUMENTS:
                 caseDetailsMap.putIfNotEmpty(
                     manageDocumentService.buildFinalApplicationBundleSupportingDocuments(caseData, NOT_SOLICITOR));
                 break;
             case COURT_BUNDLE:
-                caseDetailsMap.putIfNotEmpty(COURT_BUNDLE_LIST_KEY, manageDocumentLAService
-                    .buildCourtBundleList(caseData));
+                List<Element<HearingCourtBundle>> courtBundle = manageDocumentLAService.buildCourtBundleList(caseData);
```

Comment:
maybe rename to courtBundles as more than one

Reply:
sure